harmony-fe/src/components/appbar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react'
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import Image from 'next/image'
+import Divider from '@mui/material/Divider';
+import Link from 'next/link';
```

Comment:
Super minor but any chance we could shrink these imports by importing all of the MUI stuff from `@mui/material`? 
i.e. `import { AppBar, Box, Toolbar, ... } from '@mui/material'`